### DISCUSSION TITLE
Have you removed "WinUI 3 in UWP apps" from WinUI 3 roadmap?

### DISCUSSION THREAD

**Discussion by lighterhy** [microsoft/WindowsAppSDK#989] [Q&A]
Have you removed "WinUI 3 in UWP apps" from WinUI 3 roadmap? What is the plan for UWP apps？

**Comment by float34** [comment]
@lighterhy My understanding is that the plan is to urge developers to update existing Win32 apps, and then slowly deprecate UWP.

**Comment by ikas-mc** [comment]
They cover up, because it is really ashamed

**Comment by riverar** [comment]
Yes, it was removed. **There is no current plan for the next year**. Folks writing "UWP" apps can continue to leverage WinUI 2.x to target desktop and devices such as HoloLens and Xbox.

This is briefly covered in the latest [WinUI Community Standup (51 minutes in)](https://youtu.be/9EOKxjFyY68?t=3052).

A more in-depth discussion on Discord (below):
![image](https://user-images.githubusercontent.com/475132/125237255-3f504c00-e29a-11eb-842a-cfefba1d8dac.png)

**Reply by robloo** [reply]
AvaloniaUI is the best choice

**Comment by Lorymi** [comment]
I wish Micro$oft cared as much about UWP as it did about ASP/Web

**Comment by ccarlo88** [comment]


**Reply by robloo** [reply]
Microsoft follows the money. They never will understand that if they simply did what developers asked the money would start rolling in. They can no longer take risks and invest in the unknown. Time for smaller agile companies to take over.

**Reply by ghost** [reply]
> They can no longer take risks and invest in the unknown. 

=sAtYa nAdElLA 
the guy who cut zune , windows phone , band , mixer . basically any non-cloud products will be completely cut to keep shareholders happy, windows would've been already dead  under him if Windows 365 was not the ultimate plan. wsl2,wslg exist only because of him.

but hey he did make MS a trillion dollar company though , Money counts, no?

**Reply by robloo** [reply]
I would argue the pandemic actually saved Windows as well by showing it was still needed and useful.

But agree with your points. Max profit to the shareholders is his MO and how he got the job. But its yet to be seen if this is short term thinking. It clearly seems so to us but perhaps cloud does become so dominant in the future everything runs there. Until that time it certainly seems like he's maximizing short term profits at the expense of long-term relevance especially in the app developments space. A space which Microsoft once dominated and many of us are still invested in.

**Reply by ghost** [reply]
> Max profit to the shareholders is his MO and how he got the job. But its yet to be seen if this is short term thinking.

 ![E7e2mDCXoAQXjps](https://user-images.githubusercontent.com/88171748/127585672-c2350550-7e26-4730-bf75-c0267e04586c.jpeg)
